index.js

```diff
@@ -1,93 +1,19 @@
-const fs = require("fs"),
```

Comment:
Some of the code removed from this file is actually useful.
Please review and ensure that you have not removed any essential code.
Feel free to reach out for guidance.